gsuite_driver/main.py

```diff
@@ -31,6 +31,30 @@ def handle(event=None, context={}):
     driver_mode = config('interactive', namespace='gsuite_driver', default='True')
     environment = config('environment', namespace='gsuite_driver', default='development')
     conformance = config('conformance', namespace='gsuite_driver', default='True')
+    archive = config('archive', namespace='gsuite_driver', default='True')
```

Comment:
i expected `delete = config('delete', namespace='gsuite_driver', default='True')` due to `GSUITE_DRIVER_DELETE` env var